spec/features/job_seekers_can_subscribe_to_a_job_alert_spec.rb

```diff
@@ -2,6 +2,9 @@
 
 RSpec.feature 'A job seeker can subscribe to a job alert' do
   before { allow(EmailAlertsFeature).to receive(:enabled?) { true } }
+  let(:header) do
+    ActionView::Base.full_sanitizer.sanitize(I18n.t('subscriptions.confirmation.header'))
+  end
```

Comment:
You can avoid this by using `expect(page.html).to include(I18n.t('subscriptions.confirmation.header'))`, which is arguably a better test, because it ensures we get the structured header.

Reply:
See this.

Reply:
Weird, I thought I'd fixed that